### ISSUE TITLE
Build error in develop branch

### ISSUE THREAD

**Issue by andoh104** [autowarefoundation/autoware_ai#1032]
Build error occurred in develop branch on Cuda 9.0 ,
but  did not  occur    in master  branch on Cuda 9.0
and did not  occur    in develop branch on Cuda 8.0 .

# Bug
## Expected Behavior
No error on Cuda 9

## Actual Behavior
```
[ 9%] Building NVCC (Device) object computing/perception/localization/lib/fast_pcl/ndt_gpu/CMakeFiles/fast_pcl_ndt_gpu.dir/src/fast_pcl_ndt_gpu_generated_NormalDistributionsTransform.cu.o
In file included from /usr/local/cuda-9.0/include/common_functions.h:50:0,
from /usr/local/cuda-9.0/include/cuda_runtime.h:115,
from <command-line>:0:
/usr/local/cuda-9.0/include/crt/common_functions.h:64:24: error: token ""__CUDACC_VER__ is no longer supported. Use __CUDACC_VER_MAJOR__, __CUDACC_VER_MINOR__, and __CUDACC_VER_BUILD__ instead."" is not valid in preprocessor expressions
#define __CUDACC_VER__ "__CUDACC_VER__ is no longer supported. Use __CUDACC_VER_MAJOR__, __CUDACC_VER_MINOR__, and __CUDACC_VER_BUILD__ instead."
^
[ 9%] Linking CXX executable /home/MyProj/Autoware/ros/devel/lib/oculus_socket/oculus_sender
[ 9%] Built target oculus_sender
CMake Error at fast_pcl_ndt_gpu_generated_NormalDistributionsTransform.cu.o.cmake:207 (message):
Error generating
/home/MyProj/Autoware/ros/build/computing/perception/localization/lib/fast_pcl/ndt_gpu/CMakeFiles/fast_pcl_ndt_gpu.dir/src/./fast_pcl_ndt_gpu_generated_NormalDistributionsTransform.cu.o
computing/perception/localization/lib/fast_pcl/ndt_gpu/CMakeFiles/fast_pcl_ndt_gpu.dir/build.make:77: ターゲット 'computing/perception/localization/lib/fast_pcl/ndt_gpu/CMakeFiles/fast_pcl_ndt_gpu.dir/src/fast_pcl_ndt_gpu_generated_NormalDistributionsTransform.cu.o' のレシピで失敗しました
make[2]: *** [computing/perception/localization/lib/fast_pcl/ndt_gpu/CMakeFiles/fast_pcl_ndt_gpu.dir/src/fast_pcl_ndt_gpu_generated_NormalDistributionsTransform.cu.o] エラー 1
CMakeFiles/Makefile2:28398: ターゲット 'computing/perception/localization/lib/fast_pcl/ndt_gpu/CMakeFiles/fast_pcl_ndt_gpu.dir/all' のレシピで失敗しました
make[1]: *** [computing/perception/localization/lib/fast_pcl/ndt_gpu/CMakeFiles/fast_pcl_ndt_gpu.dir/all] エラー 2
Makefile:138: ターゲット 'all' のレシピで失敗しました
make: *** [all] エラー 2
Invoking "make -j4 -l4" failed
MyProj@USB-HDD:~/Autoware/ros$ nvcc -V
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2017 NVIDIA Corporation
Built on Fri_Sep__1_21:08:03_CDT_2017
Cuda compilation tools, release 9.0, V9.0.176
MyProj@USB-HDD:~/Autoware/ros$
```

## Steps to Reproduce the Problem

  1.`MyProj@USB-HDD:~/Autoware/ros$ ./catkin_make_release `

## Specifications

  - Ubuntu version:16.04
  - ROS Version:Kinetic
  - Autoware branch:develop

# New Feature

**Comment by andoh104** [comment]
I submit detailed log:
```
[  9%] Building NVCC (Device) object computing/perception/localization/lib/fast_pcl/ndt_gpu/CMakeFiles/fast_pcl_ndt_gpu.dir/src/fast_pcl_ndt_gpu_generated_NormalDistributionsTransform.cu.o
In file included from /usr/local/cuda-9.0/include/common_functions.h:50:0,
                 from /usr/local/cuda-9.0/include/cuda_runtime.h:115,
                 from <command-line>:0:
/usr/local/cuda-9.0/include/crt/common_functions.h:64:24: error: token ""__CUDACC_VER__ is no longer supported.  Use __CUDACC_VER_MAJOR__, __CUDACC_VER_MINOR__, and __CUDACC_VER_BUILD__ instead."" is not valid in preprocessor expressions
 #define __CUDACC_VER__ "__CUDACC_VER__ is no longer supported.  Use __CUDACC_VER_MAJOR__, __CUDACC_VER_MINOR__, and __CUDACC_VER_BUILD__ instead."
                        ^
ptxas info    : 0 bytes gmem
ptxas info    : Compiling entry function 'copyMatrixDevToDev' for 'sm_61'
ptxas info    : Function properties for copyMatrixDevToDev
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 7 registers, 368 bytes cmem[0]
Generating moc_rbsspfvehicletracker.cpp
[  9%] Built target vscan_lidar_track_automoc
Scanning dependencies of target points2costmap
[  9%] Building CXX object computing/perception/semantics/packages/object_map/CMakeFiles/points2costmap.dir/nodes/points2costmap/points2costmap.cpp.o
[ 10%] Building NVCC (Device) object computing/perception/localization/lib/fast_pcl/ndt_gpu/CMakeFiles/fast_pcl_ndt_gpu.dir/src/fast_pcl_ndt_gpu_generated_Registration.cu.o
In file included from /usr/local/cuda-9.0/include/common_functions.h:50:0,
                 from /usr/local/cuda-9.0/include/cuda_runtime.h:115,
                 from <command-line>:0:
/usr/local/cuda-9.0/include/crt/common_functions.h:64:24: error: token ""__CUDACC_VER__ is no longer supported.  Use __CUDACC_VER_MAJOR__, __CUDACC_VER_MINOR__, and __CUDACC_VER_BUILD__ instead."" is not valid in preprocessor expressions
 #define __CUDACC_VER__ "__CUDACC_VER__ is no longer supported.  Use __CUDACC_VER_MAJOR__, __CUDACC_VER_MINOR__, and __CUDACC_VER_BUILD__ instead."
                        ^
CMake Error at fast_pcl_ndt_gpu_generated_NormalDistributionsTransform.cu.o.cmake:207 (message):
  Error generating
  /home/MyProj/Autoware/ros/build/computing/perception/localization/lib/fast_pcl/ndt_gpu/CMakeFiles/fast_pcl_ndt_gpu.dir/src/./fast_pcl_ndt_gpu_generated_NormalDistributionsTransform.cu.o


computing/perception/localization/lib/fast_pcl/ndt_gpu/CMakeFiles/fast_pcl_ndt_gpu.dir/build.make:77: ターゲット 'computing/perception/localization/lib/fast_pcl/ndt_gpu/CMakeFiles/fast_pcl_ndt_gpu.dir/src/fast_pcl_ndt_gpu_generated_NormalDistributionsTransform.cu.o' のレシピで失敗しました
make[2]: *** [computing/perception/localization/lib/fast_pcl/ndt_gpu/CMakeFiles/fast_pcl_ndt_gpu.dir/src/fast_pcl_ndt_gpu_generated_NormalDistributionsTransform.cu.o] エラー 1
make[2]: *** 未完了のジョブを待っています....
Scanning dependencies of target laserscan2costmap
[ 10%] Building CXX object computing/perception/semantics/packages/object_map/CMakeFiles/laserscan2costmap.dir/nodes/laserscan2costmap/laserscan2costmap.cpp.o
CMake Error at fast_pcl_ndt_gpu_generated_Registration.cu.o.cmake:207 (message):
  Error generating
  /home/MyProj/Autoware/ros/build/computing/perception/localization/lib/fast_pcl/ndt_gpu/CMakeFiles/fast_pcl_ndt_gpu.dir/src/./fast_pcl_ndt_gpu_generated_Registration.cu.o


computing/perception/localization/lib/fast_pcl/ndt_gpu/CMakeFiles/fast_pcl_ndt_gpu.dir/build.make:84: ターゲット 'computing/perception/localization/lib/fast_pcl/ndt_gpu/CMakeFiles/fast_pcl_ndt_gpu.dir/src/fast_pcl_ndt_gpu_generated_Registration.cu.o' のレシピで失敗しました
make[2]: *** [computing/perception/localization/lib/fast_pcl/ndt_gpu/CMakeFiles/fast_pcl_ndt_gpu.dir/src/fast_pcl_ndt_gpu_generated_Registration.cu.o] エラー 1
CMakeFiles/Makefile2:28398: ターゲット 'computing/perception/localization/lib/fast_pcl/ndt_gpu/CMakeFiles/fast_pcl_ndt_gpu.dir/all' のレシピで失敗しました
make[1]: *** [computing/perception/localization/lib/fast_pcl/ndt_gpu/CMakeFiles/fast_pcl_ndt_gpu.dir/all] エラー 2
make[1]: *** 未完了のジョブを待っています....
[ 10%] Linking CXX executable /home/MyProj/Autoware/ros/devel/lib/object_map/dist_transform
[ 10%] Built target dist_transform
[ 10%] Linking CXX executable /home/MyProj/Autoware/ros/devel/lib/object_map/laserscan2costmap
[ 10%] Built target laserscan2costmap
/home/MyProj/Autoware/ros/src/computing/perception/semantics/packages/object_map/nodes/points2costmap/points2costmap.cpp:115:18: warning: ‘std::vector<int> {anonymous}::filterCostMap(std::vector<int>&)’ defined but not used [-Wunused-function]
 std::vector<int> filterCostMap(std::vector<int>& cost_map)
                  ^
[ 10%] Linking CXX executable /home/MyProj/Autoware/ros/devel/lib/object_map/points2costmap
[ 10%] Built target points2costmap
Makefile:138: ターゲット 'all' のレシピで失敗しました
make: *** [all] エラー 2
Invoking "make -j4 -l4" failed
MyProj@USB-HDD:~/Autoware/ros$ 
MyProj@USB-HDD:~/Autoware/ros$ git branch
* develop
  master
MyProj@USB-HDD:~/Autoware/ros$ nvcc -V
nvcc: NVIDIA (R) Cuda compiler driver
Copyright (c) 2005-2017 NVIDIA Corporation
Built on Fri_Sep__1_21:08:03_CDT_2017
Cuda compilation tools, release 9.0, V9.0.176
MyProj@USB-HDD:~/Autoware/ros$

```

**Comment by andoh104** [comment]
@YamatoAndo 

I tested some revisions of the develop branch, and I think this commit caused the error .
https://github.com/CPFL/Autoware/commit/cede0f32e63088eaa143e9c09e0d3d7e6ca8d7d9#diff-86c66608765cf5c93ad53f0479a58f18

```
[ 13%] Building NVCC (Device) object computing/perception/localization/lib/fast_pcl/ndt_gpu/CMakeFiles/fast_pcl_ndt_gpu.dir/src/fast_pcl_ndt_gpu_generated_MatrixHost.cu.o
ptxas info    : 0 bytes gmem
ptxas info    : Compiling entry function 'copyMatrixDevToDev' for 'sm_61'
ptxas info    : Function properties for copyMatrixDevToDev
    0 bytes stack frame, 0 bytes spill stores, 0 bytes spill loads
ptxas info    : Used 7 registers, 368 bytes cmem[0]
[ 13%] Building NVCC (Device) object computing/perception/localization/lib/fast_pcl/ndt_gpu/CMakeFiles/fast_pcl_ndt_gpu.dir/src/fast_pcl_ndt_gpu_generated_NormalDistributionsTransform.cu.o
In file included from /usr/local/cuda-9.0/include/common_functions.h:50:0,
                 from /usr/local/cuda-9.0/include/cuda_runtime.h:115,
                 from <command-line>:0:
/usr/local/cuda-9.0/include/crt/common_functions.h:64:24: error: token ""__CUDACC_VER__ is no longer supported.  Use __CUDACC_VER_MAJOR__, __CUDACC_VER_MINOR__, and __CUDACC_VER_BUILD__ instead."" is not valid in preprocessor expressions
 #define __CUDACC_VER__ "__CUDACC_VER__ is no longer supported.  Use __CUDACC_VER_MAJOR__, __CUDACC_VER_MINOR__, and __CUDACC_VER_BUILD__ instead."
                        ^
CMake Error at fast_pcl_ndt_gpu_generated_NormalDistributionsTransform.cu.o.cmake:207 (message):
  Error generating
  /home/tandoh/Autoware/ros/build/computing/perception/localization/lib/fast_pcl/ndt_gpu/CMakeFiles/fast_pcl_ndt_gpu.dir/src/./fast_pcl_ndt_gpu_generated_NormalDistributionsTransform.cu.o


computing/perception/localization/lib/fast_pcl/ndt_gpu/CMakeFiles/fast_pcl_ndt_gpu.dir/build.make:77: ターゲット 'computing/perception/localization/lib/fast_pcl/ndt_gpu/CMakeFiles/fast_pcl_ndt_gpu.dir/src/fast_pcl_ndt_gpu_generated_NormalDistributionsTransform.cu.o' のレシピで失敗しました
make[2]: *** [computing/perception/localization/lib/fast_pcl/ndt_gpu/CMakeFiles/fast_pcl_ndt_gpu.dir/src/fast_pcl_ndt_gpu_generated_NormalDistributionsTransform.cu.o] エラー 1
CMakeFiles/Makefile2:26303: ターゲット 'computing/perception/localization/lib/fast_pcl/ndt_gpu/CMakeFiles/fast_pcl_ndt_gpu.dir/all' のレシピで失敗しました
make[1]: *** [computing/perception/localization/lib/fast_pcl/ndt_gpu/CMakeFiles/fast_pcl_ndt_gpu.dir/all] エラー 2
Makefile:138: ターゲット 'all' のレシピで失敗しました
make: *** [all] エラー 2
Invoking "make -j4 -l4" failed
:~/Autoware/ros$ git branch
* (HEAD detached at cede0f3)
  develop
  master
:~/Autoware/ros$ 
```

**Comment by andoh104** [comment]
Reported on autowarefoundation/autoware_ai#4
